src/lib/targeting.test.ts

```diff
@@ -72,46 +72,82 @@ describe('shouldNotRenderEpic', () => {
 
 describe('shouldThrottle', () => {
     it('returns true if epic was viewed too recently', () => {
-        const config = { maxViewsDays: 90, maxViewsCount: 4, minDaysBetweenViews: 5 };
-        const viewLog = [{ date: new Date('2019-06-11T10:24:00').valueOf(), testId: 'A' }];
-
+        const config = { maxViewsDays: 90, maxViewsCount: 4, minDaysBetweenViews: 2 };
         const now = new Date('2019-06-12T10:24:00');
-        const got = shouldThrottle(viewLog, config, undefined, now);
-        expect(got).toBe(true);
+
+        // Epic viewed just now
+        const viewLog1 = [{ date: new Date('2019-06-12T10:23:59').valueOf(), testId: 'A' }];
+        const got1 = shouldThrottle(viewLog1, config, undefined, now);
+        expect(got1).toBe(true);
+
+        // Epic viewed yesterday
+        const viewLog2 = [{ date: new Date('2019-06-11T10:24:00').valueOf(), testId: 'A' }];
+        const got2 = shouldThrottle(viewLog2, config, undefined, now);
+        expect(got2).toBe(true);
+
+        // Epic viewed nearly 2 days ago
+        const viewLog3 = [{ date: new Date('2019-06-10T10:24:01').valueOf(), testId: 'A' }];
```

Comment:
What do you think about dropping the comment and giving the date a name? I find that descriptive variable names tend to get stale less than comments when changes are made down the line. For example:

```js
const nearlyTwoDaysAgo = new Date('2019-06-10T10:24:01').valueOf();
const viewLog3 = [{ date: nearlyTwoDaysAgo, testId: 'A' }];
```

Reply:
This is no longer needed as I've refactored the `it` blocks to only one assertion each, but this I agree this would be a good improvement.